src/components/LogIn/LogIn.js

```diff
@@ -2,6 +2,8 @@ import React from "react";
 import { useSelector } from "react-redux";
 import { isLoggedIn } from "app/selectors";
 
+import logo from "static/images/logo/logo-black-on-white.svg";
```

Comment:
Curious, why `import` the svg instead of using it just as the link in `src`?

Reply:
- Importing images that are less than 10,000 bytes returns a data URI instead of a path (excludes SVGs).
- If the file is renamed or deleted, the app will throw a compilation error
- If the file content changes in the future, Webpack will give it a different name in production so you don’t need to worry about long-term caching of assets.

Ref: https://create-react-app.dev/docs/adding-images-fonts-and-files